File: FoldSectonTableViewDemo/FoldSectonTableViewDemo/ViewController.m
//
//  ViewController.m
//  FoldSectonTableViewDemo
//
//  Created by ymq on 16/3/18.
//  Copyright © 2016年 ymq. All rights reserved.
//

#import "ViewController.h"
#import "FoldSectionTableView.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    NSDictionary *cellDataDic = @{@(0):@[@"这是一区",@"这是一区",@"这是一区",@"这是一区"], @(1):@[@"这是二区",@"这是二区",@"这是二区"], @(2):@[@"这是三区",@"这是三区",@"这是三区"]} ;
    NSArray *titleArray = @[@"一区:", @"二区", @"三区"];
   
    
    FoldSectionTableView *tableView = [[FoldSectionTableView alloc] initWithFrame:CGRectMake(0, 20, SCREEN_WIDTH, SCREEN_HEIGHT) style:UITableViewStyleGrouped];
    tableView.cellDataDic = cellDataDic;
    tableView.titleArray = titleArray;
    [self.view addSubview:tableView];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: SwiftTest/SwiftTest/ViewController.swift
//
//  ViewController.swift
//  SwiftTest
//
//  Created by ymq on 2017/11/24.
//  Copyright © 2017年 ymq. All rights reserved.
//

import UIKit

let SCREENWIDTH : CGFloat = UIScreen.main.bounds.width
let SCREENHEIGHT : CGFloat = UIScreen.main.bounds.height

typealias btnClick = (title:NSString)->Void

class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource{
     var _tableView : UITableView!
    
    
    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
        return 20;
    }
    
    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
        let cell = tableView.dequeueReusableCell(withIdentifier: "cellid")
        cell?.textLabel?.text = "\(indexPath.row)";
        
        return cell!;
    }
    
    

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        _tableView = UITableView.init(frame: CGRect.init(x: 0, y: 0, width: SCREENWIDTH, height: SCREENHEIGHT));
        _tableView.delegate = self;
        _tableView.dataSource = self;
        _tableView.register(UITableViewCell.classForCoder(), forCellReuseIdentifier: "cellid");
        
        self.view.addSubview(_tableView)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

